scheduler/src/cook/mesos/api.clj

```diff
@@ -943,6 +984,15 @@
       false
       [true {::error "must supply at least one job or instance query param"}])))
 
+(defn job-request-malformed?
+  [ctx]
+  (let [uuids (get-in ctx [:request :params :uuid])
+        allow-partial-results? (get-in ctx [:request :query-params :partial])]
+    (if uuids
+      [false {::jobs (map #(UUID/fromString %) (wrap-seq uuids))
+              ::allow-partial-results? allow-partial-results?}]
+      [true {::error "You must pass at least one job UUID."}])))
```

Comment:
Should we be enforcing this in the schema?

Reply:
We actually already are. Turns out this check was unnecessary, so I removed it. I decided to go ahead and leave this function around because it's a nice place to setup the context for the rest of the Liberator tree, and we will need it when we move the current `/list` params into `/jobs`.